src/devtools/client/debugger/src/components/Editor/StaticTooltip.tsx

```diff
@@ -11,12 +11,14 @@ type StaticTooltipProps = {
 
 export default function StaticTooltip({ targetNode, children }: StaticTooltipProps) {
   const { value: enableLargeText } = useFeature("enableLargeText");
+  const { value: inlineHitCounts } = useFeature("inlineHitCounts");
 
   return ReactDOM.createPortal(
     <div
       className={classNames(
-        "absolute z-50 flex flex-row space-x-px transform -right-1 translate-x-full bottom-4 mb-0.5 pointer-events-none",
-        enableLargeText && "bottom-6"
+        "pointer-events-none absolute bottom-4 z-50 mb-0.5 flex translate-x-full transform flex-row space-x-px",
+        enableLargeText && "bottom-6",
+        inlineHitCounts ? "-right-9" : "-right-1"
```

Comment:
This is gross 😄 but it was already gross and kind of out of scope of this PR so 🤷🏼